bin/sync_vsc_email_postfix.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+#
+# Copyright 2013-2019 Ghent University
+#
+# This file is part of vsc-administration,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# the Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/hpcugent/vsc-administration
+#
+# All rights reserved.
+#
+"""
+This script creates the list of canonical email adresses for VSC users.
+"""
+from __future__ import print_function
+
+import logging
+
+from vsc.accountpage.sync import Sync
+
+
+class VscPostfixSync(Sync):
+    CLI_OPTIONS = {
+        'postfix_alias_map': ('Location of the postfix canonical map', None, 'store', '/etc/postfix/vsc_canonical'),
+    }
+
+    def do(self, dryrun):
+        """Actual work.
+
+        - retrieve all the email addresss from changed users
+        - read the current file
+        - remove inactive users' email addresses
+        - add new/changed email addresses
+        - write file
+        """
+        active_accounts, inactive_accounts = self.get_accounts()
+
+        active_emails = dict([(a.vsc_id, a.email) for a in active_accounts])
+        address_map = dict()
+
+        inactive_emails = set([(a.vsc_id, a.email) for a in inactive_accounts])
+        try:
+            with open(self.options.postfix_alias_map, 'r') as pab:
+                address_set = set([tuple(l.split()) for l in pab.readlines() if l])
```

Comment:
strip the `@vscentum.be`? and add unittests

Reply:
We cannot add tests for things in bin?

Reply:
ofcourse we can, the `__main__` takes care of that

Reply:
https://github.ugent.be/hpcugent/vsc-data/blob/master/test/sync_gridmap.py

Reply:
That is new to me, we used to be unable to import those, afaik.

Reply:
the vsc-install 00 tests import everything we ship, since forever